test/config/config1.json

```diff
@@ -1,11 +1,12 @@
 {
-    "catalog":"test/catalog/catalog.json",
+    "catalog":"test/catalog/catalog1.json",
     "protocol":"cp",
-    "backend":"/tmp/vk",
+    "backend":"/data",
```

Comment:
please don't use custom directories, e.g. I don't have /data on my system and neither I can ask anyone else to create a special directory. That's why I suggest to use /tmp since it exists all them and does not require any privileges, i.e. everyone can write to it. Or, use local directory as you did in other config, e.g. test/data.